### PR TITLE
📖 Add kcp, Space object-vs-physical distinction to glossary

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR makes two enhancements to the glossary.  One is add some material on kcp.  The other is to distinguish between "physical space" and "space object".

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
@eaepstein
/cc @ezrasilvera 
/cc @dumb0002 
/cc @francostellari

File: docs/content/Getting-Started/user-guide.md [hunk 1]
```diff
@@ -26,7 +28,9 @@ This is a description of the concepts behind KubeStellar.
 
 **SinglePlacementSlice** - A kind of Kubernetes API object, in a Workload Description Space.  Such an object holds a list of references to `Location` & `SyncTarget` objects that match the "where predicate" of an `EdgePlacement`.  Currently there is exactly one `SinglePlacementSlice` for each `EdgePlacement` but in the future the matches for one `EdgePlacement` could be spread among several `SinglePlacementSlice` objects (analogously to `EndpointSlice` vs `Service` in Kubernetes).
 
-**Space** - A thing that behaves like a Kubernetes kube-apiserver (and the persistent storage behind it) and the subset of controllers in the kube-controller-manager that are concerned with API machinery generalities (not management of containerized workloads). A kcp logical cluster is an example. A regular Kubernetes cluster includes a space.
+**Space Object** - A Kubernetes-style API object that represents a physical space.  Its "kind" is `Space`, so we include the "object" part when writing specifically about the API objects to clearly distinguish them from what they represent.
```
Comment (ezrasilvera) [inline docs/content/Getting-Started/user-guide.md:31]: nit: on all other objects you wrote "A kind of Kubernetes API object..."  why not using the same here, it imply it's different than the others. Is it ?

**Comment by MikeSpreitzer** [comment]
The force-push to ac9964f0 updates the description of "Space Object" in response to the review remark above.

**Comment by clubanderson** [comment]
what is 'physical' about a 'physical space'?

File: docs/content/Getting-Started/user-guide.md [hunk 2]
```diff
@@ -26,7 +28,9 @@ This is a description of the concepts behind KubeStellar.
...
 
-**Space** - A thing that behaves like a Kubernetes kube-apiserver (and the persistent storage behind it) and the subset of controllers in the kube-controller-manager that are concerned with API machinery generalities (not management of containerized workloads). A kcp logical cluster is an example. A regular Kubernetes cluster includes a space.
+**Space Object** - A kind of Kubernetes API object that _represents_ a physical space.  Its "kind" is `Space`, so we include the "object" part when writing specifically about the API objects to clearly distinguish them from what they represent.
+
+**Space**, **physical space** (use the latter term to clearly distinguish from "space object") - A thing that behaves like a Kubernetes kube-apiserver (including the persistent storage behind it) and the subset of controllers in the kube-controller-manager that are concerned with API machinery generalities (not management of containerized workloads). A kcp logical cluster is an example. A regular Kubernetes cluster includes a space.
```
Comment (ezrasilvera) [inline docs/content/Getting-Started/user-guide.md:34]: why can't we say A kcp logical cluster is an example a regular Kubernetes cluster is another example (although it supports additional capabilities) . I find the "include a space" confusing

**Comment by MikeSpreitzer** [comment]
The force-push to f1bdc0c1 responds to @ezrasilvera's review above.

**Pull request merged**